Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: all dockerfiles
 all:
```

Comment:
Not really related to this PR but do we need the `all` target?

Reply:
In my opinion, `make` without arguments on this repository is not expected to update auto-generated dockerfiles.